autoload/javacomplete/util.vim

```diff
@@ -350,14 +350,23 @@ function! javacomplete#util#GetBase(extra)
   return base
 endfunction
 
+function! javacomplete#util#RemoveEmptyClasses(classes)
+  let newClasses = a:classes[:]
+  let emptyClassIndex = index(newClasses, '')
+  if emptyClassIndex > -1
```

Comment:
What if there are more than 1 empty strings? May be it's better to use `while` loop here?

Reply:
I discover the `filter` function and use it instead of using `index` and `unlet`.